Resources/templates/responsive/creator/partials/subscription_item.php

```diff
@@ -18,8 +18,12 @@
     </div>
     <div class="card-footer">
         <div class="reward-donate text-center">
-            <a class="btn btn-lg btn-cyan text-uppercase" href="/invest/<?= $project->id ?>/payment?reward=<?= $subscription->id ?>">
-                <?= $this->text('project-regular-support') ?>
+            <a class="btn btn-lg btn-cyan text-uppercase" href="/invest/<?= $project->id ?>/<?= $subscription->id ?>">
```

Comment:
I think that when fixing the conflicts you missed this -> https://github.com/GoteoFoundation/goteo/pull/591/files